screens/SelectSignUpTypeScreen.tsx

```diff
@@ -0,0 +1,101 @@
+import { Alert, ImageBackground, Pressable, StyleSheet } from 'react-native';
+
+import { Text, View } from '../components/Themed';
+import Svg, { SvgUri } from 'react-native-svg';
+import { RootStackParamList } from '../types';
+import { NativeStackNavigationProp } from '@react-navigation/native-stack';
+import SetupWrapper from '../components/SetupWrapper';
+import React from 'react';
+
+type SelectSignUpTypeScreenProp = NativeStackNavigationProp<
+  RootStackParamList,
+  'SelectSignUpTypeScreen'
+>;
+
+export interface SelectSignUpTypeScreenProps {
+  navigation: SelectSignUpTypeScreenProp;
+}
+
+export default function SelectSignUpTypeScreen({
+  navigation: { navigate },
+}: SelectSignUpTypeScreenProps) {
+  return (
+    <SetupWrapper>
+      <View style={styles.main}>
+        <Pressable
+          style={styles.blueButton}
+          onPress={() => {
+            //@ts-expect-error
+            navigate('SignUpScreen', {
+              Type: 'Student',
+            });
+          }}
+        >
+          <SvgUri
+            uri={'https://icons.getbootstrap.com/assets/icons/person.svg'}
+            height={100}
+            width={100}
+            fill={'#ffffff'}
+            style={styles.icon}
+          />
+
+          <Text style={styles.boxText}>
+            Ik ben een <Text style={styles.boldText}>Student</Text>
+          </Text>
+        </Pressable>
+
+        <Pressable
+          style={styles.blueButton}
+          onPress={() => {
+            //@ts-expect-error
+            navigate('SignUpScreen', {
+              Type: 'Company',
+            });
+          }}
+        >
+          <SvgUri
+            uri={'https://icons.getbootstrap.com/assets/icons/building.svg'}
```

Comment:
Should we maybe download this asset and import it into the project, or was that too much of a PITA?

Reply:
way too much of a pain in the ass for current project imo, although it goes against my goal its just hassle for something simple